src/Primitives/emulated.cpp

```diff
@@ -365,7 +367,7 @@ def_prim(http_post, tenToOneU32) {
     uint32_t authorization = arg3.uint32;
     uint32_t authorization_len = arg2.uint32;
     int32_t response = arg1.uint32;
-    uint32_t size = arg0.uint32;
+    // uint32_t size = arg0.uint32; // never used in emulator
```

Comment:
we can just remove these lines

Reply:
The general idea was that if we needed them in the emulator in the future then all arguments would still be there and not just some of them with some gaps in between.